### ISSUE TITLE
Captchas not working

### ISSUE THREAD

**Issue by khooniatma** [FlareSolverr/FlareSolverr#906]
### Have you checked our README?

- [X] I have checked the README

### Have you followed our Troubleshooting?

- [X] I have followed your Troubleshooting

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you checked the discussions?

- [X] I have read the Discussions

### Environment

```markdown
- FlareSolverr version: 3.3.4
- Last working FlareSolverr version: 3.3.4
- Operating system: Windows
- Are you using Docker: [yes/no] no
- FlareSolverr User-Agent (see log traces or / endpoint):
- Are you using a VPN: [yes/no] no
- Are you using a Proxy: [yes/no] no
- Are you using Captcha Solver: [yes/no] no
- If using captcha solver, which one: no
- URL to test this issue:
```

### Description

I am seeing the same issue as https://github.com/FlareSolverr/FlareSolverr/issues/811

That issue states that the issue is fixed with 3.3.0 release but I am consistently seeing it on 3.3.4. 

Logs below - 

```
2023-09-16 11:38:50 INFO     ReqId 13240 FlareSolverr 3.3.4
2023-09-16 11:38:50 DEBUG    ReqId 13240 Debug log enabled
2023-09-16 11:38:50 INFO     ReqId 13240 Testing web browser installation...
2023-09-16 11:38:50 INFO     ReqId 13240 Platform: Windows-10-10.0.22621-SP0
2023-09-16 11:38:50 INFO     ReqId 13240 Chrome / Chromium path: C:\Users\Administrator\Documents\flaresolverr_windows_x64\flaresolverr\chrome\chrome.exe
2023-09-16 11:38:50 INFO     ReqId 13240 Chrome / Chromium major version: 115
2023-09-16 11:38:50 INFO     ReqId 13240 Launching web browser...
2023-09-16 11:38:50 DEBUG    ReqId 13240 Launching web browser...
2023-09-16 11:38:54 DEBUG    ReqId 13240 Started executable: `C:\Users\Administrator\appdata\roaming\undetected_chromedriver\undetected_chromedriver.exe` in a child process with pid: 16936
2023-09-16 11:38:54 INFO     ReqId 13240 FlareSolverr User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.0.0 Safari/537.36
2023-09-16 11:38:54 INFO     ReqId 13240 Test successful!
2023-09-16 11:38:54 INFO     ReqId 13240 Serving on http://0.0.0.0:8191
2023-09-16 11:42:01 INFO     ReqId 6708 Incoming request => POST /v1 body: {'maxTimeout': 150000, 'cmd': 'request.get', 'url': 'https://bitsearch.nocensor.cloud/search?sort=date&order=desc&limit=100'}
2023-09-16 11:42:01 DEBUG    ReqId 6708 Launching web browser...
2023-09-16 11:42:01 DEBUG    ReqId 6708 Started executable: `C:\Users\Administrator\appdata\roaming\undetected_chromedriver\chromedriver.exe` in a child process with pid: 18352
2023-09-16 11:42:02 DEBUG    ReqId 6708 New instance of webdriver has been created to perform the request
2023-09-16 11:42:02 DEBUG    ReqId 12592 Navigating to... https://bitsearch.nocensor.cloud/search?sort=date&order=desc&limit=100
2023-09-16 11:42:03 DEBUG    ReqId 12592 Response HTML:
<html lang="en-US" class="lang-en"><head><title>Just a moment...</title><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta http-equiv="X-UA-Compatible" content="IE=Edge"><meta name="robots" content="noindex,nofollow"><meta name="viewport" content="width=device-width,initial-scale=1"><link href="/cdn-cgi/styles/challenges.css" rel="stylesheet"><script src="/cdn-cgi/challenge-platform/h/b/orchestrate/chl_page/v1?ray=8076f49faed318fd"></script><script src="https://challenges.cloudflare.com/turnstile/v0/b/8370c0b3/api.js?onload=wcgW6&amp;render=explicit" async="" defer="" crossorigin="anonymous"></script></head><body class="no-js"><div class="main-wrapper" role="main"><div class="main-content"><h1 class="zone-name-title h1">bitsearch.nocensor.cloud</h1><h2 id="challenge-running" class="h2">Checking if the site connection is secure</h2><div id="challenge-stage" style="display: block;"><div id="turnstile-wrapper" class="captcha-prompt spacer"><div><iframe src="https://challenges.cloudflare.com/cdn-cgi/challenge-platform/h/b/turnstile/if/ov2/av0/rcv0/0/satfl/0x4AAAAAAADnPIDROrmt1Wwj/light/normal" allow="cross-origin-isolated; fullscreen" sandbox="allow-same-origin allow-scripts allow-popups" id="cf-chl-widget-satfl" tabindex="0" title="Widget containing a Cloudflare security challenge" style="border: none; overflow: hidden; width: 300px; height: 65px;"></iframe><input type="hidden" name="cf-turnstile-response" id="cf-chl-widget-satfl_response"></div></div></div><div id="challenge-spinner" class="spacer loading-spinner" style="display: none; visibility: hidden;"><div class="lds-ring"><div></div><div></div><div></div><div></div></div></div><div id="challenge-body-text" class="core-msg spacer">bitsearch.nocensor.cloud needs to review the security of your connection before proceeding.</div><div id="challenge-success" style="display: none;"><div class="h2"><span class="icon-wrapper"><img class="heading-icon" alt="Success icon" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADQAAAA0CAMAAADypuvZAAAANlBMVEUAAAAxMTEwMDAxMTExMTEwMDAwMDAwMDAxMTExMTExMTEwMDAwMDAxMTExMTEwMDAwMDAxMTHB9N+uAAAAEXRSTlMA3zDvfyBAEJC/n3BQz69gX7VMkcMAAAGySURBVEjHnZZbFoMgDEQJiDzVuv/NtgbtFGuQ4/zUKpeMIQbUhXSKE5l1XSn4pFWHRm/WShT1HRLWC01LGxFEVkCc30eYkLJ1Sjk9pvkw690VY6k8DWP9OM9yMG0Koi+mi8XA36NXmW0UXra4eJ3iwHfrfXVlgL0NqqGBHdqfeQhMmyJ48WDuKP81h3+SMPeRKkJcSXiLUK4XTHCjESOnz1VUXQoc6lgi2x4cI5aTQ201Mt8wHysI5fc05M5c81uZEtHcMKhxZ7iYEty1GfhLvGKpm+EYkdGxm1F5axmcB93DoORIbXfdN7f+hlFuyxtDP+sxtBnF43cIYwaZAWRgzxIoiXEMESoPlMhwLRDXeK772CAzXEdBRV7cmnoVBp0OSlyGidEzJTFq5hhcsA5388oSGM6b5p+qjpZrBlMS9xj4AwXmz108ukU1IomM3ceiW0CDwHCqp1NjAqXlFrbga+xuloQJ+tuyfbIBPNpqnmxqT7dPaOnZqBfhSBCteJAxWj58zLk2xgg+SPGYM6dRO6WczSnIxxwEExRaO+UyCUhbOp7CGQ+kxSUfNtLQFC+Po29vvy7jj4y0yAAAAABJRU5ErkJggg=="></span>Connection is secure</div><div class="core-msg spacer">Proceeding...</div></div><noscript><div id="challenge-error-title"><div class="h2"><span class="icon-wrapper"><div class="heading-icon warning-icon"></div></span><span id="challenge-error-text">Enable JavaScript and cookies to continue</span></div></div></noscript></div></div><script>(function(){window._cf_chl_opt={cvId: '2',cZone: "bitsearch.nocensor.cloud",cType: 'managed',cNounce: '2460',cRay: '8076f49faed318fd',cHash: 'dcd88581265a125',cUPMDTk: "\/search?sort=date&order=desc&limit=100&__cf_chl_tk=3vCA1M5s9e9t2DHqy9coLNvDY6LhSLtlk0S8ZpZb0Jc-1694844723-0-gaNycGzNDVA",cFPWv: 'b',cTTimeMs: '1000',cMTimeMs: '0',cTplV: 5,cTplB: 'cf',cK: "visitor-time",fa: "\/search?sort=date&order=desc&limit=100&__cf_chl_f_tk=3vCA1M5s9e9t2DHqy9coLNvDY6LhSLtlk0S8ZpZb0Jc-1694844723-0-gaNycGzNDVA",md: "_Psrz2MzvBcL4nHsysihQHt3JJ7HOnWutgboHcSzLv0-1694844723-0-AZ05VIi-rWBjI7auwfsThYA9Wwv8MwDaFUtKwfW5ssr-PJEGIC73kgl1Z6tUoW2fad8_iPQFlkGyum4xiW7bQg0glE0ZJYD577otiUHXHYkan8AHiD_maZTU9KeWU3hM-nXQL2pLKwkbFXhJKn0M5LjAPREPlv_SyXQOk7VK7bnbgveHl-hBg0KJ21m9VgPl6kU8OMAnVSitldQnmA9oPGmhmPfo2OWQnOu1nl92B7Wlo2IrjrMDitN4rD0YGTcZTbcq-7SZ8qSZybO0tqooPTlS8jo24oEhrftV-L6MlaU8Dqnc7wgn6UHdj5eGYduqvkwQhN9UlmMenPmoGqXHP7Z317AJGzjHaK2dnvmQWdh2OLKXFgQQME0Ym3m6pwFoSmyLUDOCsnbpOTub9ylX1Wh-i-ik6L6UD3hSlrFpffAWGPZTzpC6BhxWs2Zabj_WVBsq1SpfUQrM94Imlscwaen5xx-kPoi6Jv4-GrY02MmlUr6r5JiTPkdVGfZWUx1YPk0TPGlVaiPD9qlRVdf1y39McuHMzcziH9bMwNHk3SkaK_WjmMFvxsvtzAcqlnVUdfi1SNigwZs5Mu0RXgWpxaUDNIS9suNGVvE2V8y0ONSiaJvGZEykLJfo0EsckCc-f-DX6DbOQ3ranL6KPTmxwt8eCw_CHRTBY6ft9CUKPhjCPrFn9721xgG99schKwzjNSAXDuQkwVKpTEqj1qB6v7A1AIKwiqP-nQO5rlolqWdB1eyiuMoQjMZyoDzsNe2QwmF7j88sz9_eMzsXOxgpWSry2D10e0FJBMEnajz_LY6f4f4n-wCPFH9Jz_jdnroI6pUUeWiUZ7fqz5sk2gZsD5NGRCQPpcMaML4inU3u_SJqCSOZZCNlHk3aRP7MqA-5r--OZy6FMj1GqIhmrSIYXzR40tdWozs6DJA_S5rNPH3c7kb9x7kYt3bGMhlZ1S31F7vIH9oU7SzDqxx7bLk2zqb3Gg1r2Ip3J4CurOd0CKgmDPC5V5vtbBTAyG8C7Yl-Azl6-9nBBE_2dHp2dxDS3CS6zvahcKneYvgOT8V2-olRgYv7y0CmprjiIyL-wLgXJbEDMk6PvN5QqCcxd95DXIdNcJ_tPHF1vzAav81meZ4zRGYnPffC_7vfYgOSO13FS1stanwpxJ-QAQCHBhOyeVR8AEgRIunvUMSGmMErA2BCIPvzQFc0g5OYnyESGrKVxYY6PauNheub8hBkf5QTJyIPTnc2coqO4OknsWPRWd9mlGJOj_JIxnB8g0T-ap7G7h6P-7hwlnzoimJJl7uKp-on7O5tvUGeCNxWulEV2mXXbO53vAtE7c4seq5EQnzidhzC4SDaWHl4rAB-ofkcGLpewj6MFpBbeUxL7lcpsWqvgM5yjVc2ZmucBiti8bkyS3Gq9lGId5huA7_CvetIViu14-95RG2ZcvSQBKZxis_gSZl6DCO0Wr7aLk5d_zyDqLmJ-DQIww7TggsID99Q2kj6r-IJQ_jdIXUe8qzRAM-fHTVpDRjjZu9KV7BFyGI1-3WNRd98eYAmYQ8GlFL0JdlFQ5eOl4s8HEsyKDN3akTZTLjV2WTC94e1nFjzUYatta1kc1Wka50wKF4EQqCb_ncvi_r7oyMgaZTTW7zerER78vaX1WbPZwNLNUQjBHwO9J_n_yyKJHzo1QFUEllRHHFDTUigVimKtFtPtOKecKXcbhjtxXqTfXcaAlGUUzmFKlW65AK3_h84gSXhT3R-J3O4_cRwE4qi4rPW_MpjESCnPqM4TIjGloiZOgcZVUJVj9Ka7sHIkKdsKgol6gqcnob27lXO9qGkUfKXbAskhdcHh0vbc8qfjnd0Sqe-Ed8ebTNs8aiqNF7Kbku9SlNDiffMWlcaFrmACTaY9PmeQ7tIv3Q-Bnl-F-eRzvxL22TVB5PvYYxovX3ddULUDn_Z5S7TnBb0YjhAvcNf0ddVwrOUE7_1coNC1pzh2K50SVzTv6OVfrVDjabJa-Ib7xqnEa00-x8sZL6IybzD4uy_Dj5uRVaZMsEtoUHw7hKKaQ8o-72JMD0oMKiRunBu_MQAcnCWi3a_TdhrtFHQQQVRgBfht5RC47PXK7_LMr4G1HAqD6XlgEo-l1Oo1qd7zNcXjCajPdGhDImEIvieeo6lzJOcn3XacCz02mvYHreryOT8jkhJwm-aaVr581ObPZZe4xOnplSxlmKuuyQtwimtE1yNtiu5MEak0aSyk8xpuODyoDiv1gy7UoYFuUNHLH-pA4Dneb6yZgCn1Om1F75G_E06JO9k_-IC6rDs1LC2lj2CH812NPM1cCHDAu5nFyz1m4fY8owo5Ti8JuXYi30kRz1Q3_6G1ut5Wz7F2M8ZyFsJCclPGRGImaH0gDI9ZoX1ZPwzLQuNXHXqeXC2z0Iwk0w6_8EzADCdIrYo__ETaY8hU4K3ovtxSiblrqtedSG_fJlbdfICmEJRj_5Icnpl6H3rlQve045BA-doaJyEJvwa9FMMrfHIMmzRPtZIEDVFbRHmr280FSRYu-rlFEHn-sNtfyHKlUV47V7hCEILiMizFTHxD5g6MiEAIPP5AAGcixK-d8iONdjcMJ2T--U8OCjYVmx0PnxbUfVDIzo9Bl76Vj1uR1GJ77-3x5_qC6h83dev_W9AYQdGAeTq8uI0FO8X8gUZqra_6Iqs8N40sCGpCJ9IHlEtjWgXN0208_Aj94q1M3orUEqbGTwQS_tSaeUuo1R0ClTQ3HtV0TPziM4x_mAgitsfLpxKrqjxNGK_4qe5ABa7ZWWKgH1Pfn8-ox6feX85FbqeWk0pFCPEB0SPi9L0V6fb8R4T659poGv6S8EQ4fm-8HulwZ9XTEMENr5wmYA-Fj9Q1RvbTl1KCbyCbNcWpkL86dQ3u_8c8R6eMd7x-YpIBTGi8as3m_Z_ToL6GTfws3AGxWq-X7WH08AyOhRd6da5SqnA_2YAeHtSc3LUHXAc_ee7vY-5EH4pSKIpP9dlzTpBXv5RX3k4WuW0yj2Q0BP7fCh9CbeLkXMw2nmw62KIP2CI2pBrDvDmy7wZlh9tRo_au0Mp4yNyjdOjd3qbXKROEIrRyil7n5380fHeROyyOhqJsolInqKxjRwpUrV8mKQOuRRaRGbd9iKHpb-0AH0VG2Fq5XSEaWZuUIjwEIP0zPOaQQPliRLRb7rGhIUjlmQxY1ZJV8Aw09T9Zc-WfTfMqiIQXrU6spRFnEk2b7Hr811tdO89HbSuWNL8ayp4w2lrWwFo9CbI_TgraxQX0X6a_SsMMBXIEoq9pinJbespjGRqfT-COHwcKuzaFK-wZn_v4H0JnUql5Gaux0b1IHzrkEl-NeidjQ93AGk",cRq: {ru: 'aHR0cHM6Ly9iaXRzZWFyY2gubm9jZW5zb3IuY2xvdWQvc2VhcmNoP3NvcnQ9ZGF0ZSZvcmRlcj1kZXNjJmxpbWl0PTEwMA==',ra: 'TW96aWxsYS81LjAgKFdpbmRvd3MgTlQgMTAuMDsgV2luNjQ7IHg2NCkgQXBwbGVXZWJLaXQvNTM3LjM2IChLSFRNTCwgbGlrZSBHZWNrbykgQ2hyb21lLzExNS4wLjAuMCBTYWZhcmkvNTM3LjM2',rm: 'R0VU',d: 'tPUsYfh4NekaRuGqFrouCIezgojkMKh9Te24yW02q69CLSkMlScNRSuwhHxyG7EqmTf3KKY4oE41HSrgxlVLXfKxWXkS3XQ4WCrhiSn/hfiQuK3o47xUo2u9OrvIP1YGE9GGO4FneXJPIRnA6l00BNXtUY54AZIPVbaOI2rWYRA1n5QnIfPVs3H7EZv4ArAJ31Etw+UUeJbQk4bEYuS88lWWm+mvRJ3nwXrFD2hAloosZ095jb+AR6lozVwU/OAXddC0PWh0t5GBf4Mowe+mm5K3A7P8JKskMVOuFqTlFOi2jyeVyoB3H/HABntDbJvYB9GO6ekfPdiSgz3c3Lug+j+q9GS2BRwgXJ7OK38rGalIifNuZs2+wA3WeT23hUy14vaLfczxaw7V+zNfhp+V/nGqzTqgFl1OQqXB6dsgCcoVzgyK8z156rF5hkHgp5Ap41k6Isu5Kjkb62tN0GHRvLiGMQY9vMot30vD2rRMgb66tkz+BJeQjbzo204rUv5p',t: 'MTY5NDg0NDcyMy4xNDIwMDA=',cT: Math.floor(Date.now() / 1000),m: 'qIikiG+eh0Ll4fIpObiGW831XQY0qyywavjg5J7SqkM=',i1: '/Dw3aPwpr3aVU1uVgN0JAg==',i2: 'Hi8Vu9nDHwSaKwfdwt8F+w==',zh: 'Ix3eDZk0I8apnSZg8sxu0wopEDB3Y+J1lgrbErRMerI=',uh: '49m45payZ+JaK1qP7bVg0G3ztg3QEguIG+n0xYKxzGA=',hh: 'TuZ4m5lntSK5i4+xlRtSZg4YIaTHaMkRfhr4FllqIWY=',}};var cpo = document.createElement('script');cpo.src = '/cdn-cgi/challenge-platform/h/b/orchestrate/chl_page/v1?ray=8076f49faed318fd';window._cf_chl_opt.cOgUHash = location.hash === '' && location.href.indexOf('#') !== -1 ? '#' : location.hash;window._cf_chl_opt.cOgUQuery = location.search === '' && location.href.slice(0, location.href.length - window._cf_chl_opt.cOgUHash.length).indexOf('?') !== -1 ? '?' : location.search;if (window.history && window.history.replaceState) {var ogU = location.pathname + window._cf_chl_opt.cOgUQuery + window._cf_chl_opt.cOgUHash;history.replaceState(null, null, "\/search?sort=date&order=desc&limit=100&__cf_chl_rt_tk=3vCA1M5s9e9t2DHqy9coLNvDY6LhSLtlk0S8ZpZb0Jc-1694844723-0-gaNycGzNDVA" + window._cf_chl_opt.cOgUHash);cpo.onload = function() {history.replaceState(null, null, ogU);}}document.getElementsByTagName('head')[0].appendChild(cpo);}());</script><div class="footer" role="contentinfo"><div class="footer-inner"><div class="clearfix diagnostic-wrapper"><div class="ray-id">Ray ID: <code>8076f49faed318fd</code></div></div><div class="text-center" id="footer-text">Performance &amp; security by <a rel="noopener noreferrer" href="https://www.cloudflare.com?utm_source=challenge&amp;utm_campaign=m" target="_blank">Cloudflare</a></div></div></div></body></html>
2023-09-16 11:42:03 INFO     ReqId 12592 Challenge detected. Title found: Just a moment...
2023-09-16 11:42:03 DEBUG    ReqId 12592 Waiting for title (attempt 1): Just a moment...
2023-09-16 11:42:04 DEBUG    ReqId 12592 Timeout waiting for selector
2023-09-16 11:42:04 DEBUG    ReqId 12592 Try to find the Cloudflare verify checkbox...
2023-09-16 11:42:04 DEBUG    ReqId 12592 Cloudflare verify checkbox not found on the page.
2023-09-16 11:42:04 DEBUG    ReqId 12592 Try to find the Cloudflare 'Verify you are human' button...
2023-09-16 11:42:04 DEBUG    ReqId 12592 The Cloudflare 'Verify you are human' button not found on the page.
2023-09-16 11:42:06 DEBUG    ReqId 12592 Waiting for title (attempt 2): Just a moment...
2023-09-16 11:42:07 DEBUG    ReqId 12592 Timeout waiting for selector
2023-09-16 11:42:07 DEBUG    ReqId 12592 Try to find the Cloudflare verify checkbox...
2023-09-16 11:42:07 DEBUG    ReqId 12592 Cloudflare verify checkbox not found on the page.
2023-09-16 11:42:07 DEBUG    ReqId 12592 Try to find the Cloudflare 'Verify you are human' button...
2023-09-16 11:42:07 DEBUG    ReqId 12592 The Cloudflare 'Verify you are human' button not found on the page.
2023-09-16 11:42:09 DEBUG    ReqId 12592 Waiting for title (attempt 3): Just a moment...
2023-09-16 11:42:10 DEBUG    ReqId 12592 Timeout waiting for selector
2023-09-16 11:42:10 DEBUG    ReqId 12592 Try to find the Cloudflare verify checkbox...
2023-09-16 11:42:11 DEBUG    ReqId 12592 Cloudflare verify checkbox not found on the page.
2023-09-16 11:42:11 DEBUG    ReqId 12592 Try to find the Cloudflare 'Verify you are human' button...
2023-09-16 11:42:11 DEBUG    ReqId 12592 The Cloudflare 'Verify you are human' button not found on the page.
2023-09-16 11:42:13 DEBUG    ReqId 12592 Waiting for title (attempt 4): Just a moment...
2023-09-16 11:42:14 DEBUG    ReqId 12592 Timeout waiting for selector
2023-09-16 11:42:14 DEBUG    ReqId 12592 Try to find the Cloudflare verify checkbox...
2023-09-16 11:42:14 DEBUG    ReqId 12592 Cloudflare verify checkbox not found on the page.
2023-09-16 11:42:14 DEBUG    ReqId 12592 Try to find the Cloudflare 'Verify you are human' button...
2023-09-16 11:42:14 DEBUG    ReqId 12592 The Cloudflare 'Verify you are human' button not found on the page.
2023-09-16 11:42:16 DEBUG    ReqId 12592 Waiting for title (attempt 5): Just a moment...
2023-09-16 11:42:18 DEBUG    ReqId 12592 Timeout waiting for selector
2023-09-16 11:42:18 DEBUG    ReqId 12592 Try to find the Cloudflare verify checkbox...
2023-09-16 11:42:18 DEBUG    ReqId 12592 Cloudflare verify checkbox not found on the page.
2023-09-16 11:42:18 DEBUG    ReqId 12592 Try to find the Cloudflare 'Verify you are human' button...
2023-09-16 11:42:18 DEBUG    ReqId 12592 The Cloudflare 'Verify you are human' button not found on the page.
2023-09-16 11:42:20 DEBUG    ReqId 12592 Waiting for title (attempt 6): Just a moment...
2023-09-16 11:42:21 DEBUG    ReqId 12592 Timeout waiting for selector
2023-09-16 11:42:21 DEBUG    ReqId 12592 Try to find the Cloudflare verify checkbox...
2023-09-16 11:42:21 DEBUG    ReqId 12592 Cloudflare verify checkbox not found on the page.
2023-09-16 11:42:21 DEBUG    ReqId 12592 Try to find the Cloudflare 'Verify you are human' button...
2023-09-16 11:42:21 DEBUG    ReqId 12592 The Cloudflare 'Verify you are human' button not found on the page.
2023-09-16 11:42:23 DEBUG    ReqId 12592 Waiting for title (attempt 7): Just a moment...
2023-09-16 11:42:25 DEBUG    ReqId 12592 Timeout waiting for selector
2023-09-16 11:42:25 DEBUG    ReqId 12592 Try to find the Cloudflare verify checkbox...
2023-09-16 11:42:25 DEBUG    ReqId 12592 Cloudflare verify checkbox not found on the page.
2023-09-16 11:42:25 DEBUG    ReqId 12592 Try to find the Cloudflare 'Verify you are human' button...
2023-09-16 11:42:25 DEBUG    ReqId 12592 The Cloudflare 'Verify you are human' button not found on the page.
2023-09-16 11:42:27 DEBUG    ReqId 12592 Waiting for title (attempt 8): Just a moment...
2023-09-16 11:42:28 DEBUG    ReqId 12592 Timeout waiting for selector
2023-09-16 11:42:28 DEBUG    ReqId 12592 Try to find the Cloudflare verify checkbox...
2023-09-16 11:42:28 DEBUG    ReqId 12592 Cloudflare verify checkbox not found on the page.
2023-09-16 11:42:28 DEBUG    ReqId 12592 Try to find the Cloudflare 'Verify you are human' button...
2023-09-16 11:42:28 DEBUG    ReqId 12592 The Cloudflare 'Verify you are human' button not found on the page.
2023-09-16 11:42:30 DEBUG    ReqId 12592 Waiting for title (attempt 9): Just a moment...
2023-09-16 11:42:31 DEBUG    ReqId 12592 Timeout waiting for selector
2023-09-16 11:42:31 DEBUG    ReqId 12592 Try to find the Cloudflare verify checkbox...
2023-09-16 11:42:31 DEBUG    ReqId 12592 Cloudflare verify checkbox not found on the page.
2023-09-16 11:42:31 DEBUG    ReqId 12592 Try to find the Cloudflare 'Verify you are human' button...
2023-09-16 11:42:31 DEBUG    ReqId 12592 The Cloudflare 'Verify you are human' button not found on the page.
2023-09-16 11:42:33 DEBUG    ReqId 12592 Waiting for title (attempt 10): Just a moment...
2023-09-16 11:42:34 DEBUG    ReqId 12592 Timeout waiting for selector
2023-09-16 11:42:34 DEBUG    ReqId 12592 Try to find the Cloudflare verify checkbox...
2023-09-16 11:42:34 DEBUG    ReqId 12592 Cloudflare verify checkbox not found on the page.
2023-09-16 11:42:34 DEBUG    ReqId 12592 Try to find the Cloudflare 'Verify you are human' button...
2023-09-16 11:42:34 DEBUG    ReqId 12592 The Cloudflare 'Verify you are human' button not found on the page.
2023-09-16 11:42:36 DEBUG    ReqId 12592 Waiting for title (attempt 11): Just a moment...
```

### Logged Error Messages

```text
2023-09-16 11:44:31 DEBUG    ReqId 12592 Cloudflare verify checkbox not found on the page.
2023-09-16 11:44:31 DEBUG    ReqId 12592 Try to find the Cloudflare 'Verify you are human' button...
2023-09-16 11:44:31 DEBUG    ReqId 12592 The Cloudflare 'Verify you are human' button not found on the page.
2023-09-16 11:44:32 DEBUG    ReqId 6708 A used instance of webdriver has been destroyed
2023-09-16 11:44:32 ERROR    ReqId 6708 Error: Error solving the challenge. Timeout after 150.0 seconds.
2023-09-16 11:44:32 DEBUG    ReqId 6708 Response => POST /v1 body: {'status': 'error', 'message': 'Error: Error solving the challenge. Timeout after 150.0 seconds.', 'startTimestamp': 1694844721279, 'endTimestamp': 1694844872542, 'version': '3.3.4'}
2023-09-16 11:44:32 INFO     ReqId 6708 Response in 151.263 s
2023-09-16 11:44:32 INFO     ReqId 6708 127.0.0.1 POST http://127.0.0.1:8191/v1 500 Internal Server Error
```

**Comment by khooniatma** [comment]
I also saw this issue https://github.com/FlareSolverr/FlareSolverr/issues/881 reported the same and states that it was fixed in v3.3.4, but I am on this version and it doesn't work on Windows

**Comment by khooniatma** [comment]
Updated v3.3.6 and it worked.
